src/dotnet/commands/dotnet-new/CSharp_xunittest/project.json.template

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.0.0-*",
+
+    "dependencies": {
+        "Microsoft.NETCore.App": {
+            "type": "platform",
+            "version": "1.0.0-rc2-23928"
+        },
+        "System.Runtime.Serialization.Primitives": "4.1.1-rc2-23928",
+        "xunit": "2.1.0",
+        "dotnet-test-xunit": "1.0.0-*"
+    },
+
+    "testRunner":  "xunit",
+
+    "frameworks": {
+        "netstandardapp1.5": {
```

Comment:
@Petermarcu actually, shouldn't this be `netstandard1.5`? Unit tests are actually just class libraries, right?